README.md

```diff
@@ -1422,6 +1422,27 @@ features in the plugin configuration
 The list of privileges along with their description can be found in
 [security privileges](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html).
 
+## Configuration - Elasticsearch Output Data Stream
```

Comment:
Could you add this section in table of contents?
ToC is here under this section:
https://github.com/uken/fluent-plugin-elasticsearch/blob/master/README.md#fluentpluginelasticsearch-a-plugin-for-fluentd

Reply:
I'll fix it.